packages/bits/spec/virtual-camera/PercyLens.ts

```diff
@@ -24,6 +24,8 @@ export class PercyLens implements ILens {
 
     public async cameraOFF() {}
 
+    public toolConfig() { console.warn("No config is availalbe for Percy"); }
+
     private checkSettings() {
         this.percyConfig.widths = [...this.settings.responsive];
```

Comment:
Maybe `cameraSettings` to make it more clear.

```suggestion
        this.percyConfig.widths = [...this.cameraSettings.responsive];
```